### ISSUE TITLE
Fix rare race condition with b2 integration

### ISSUE THREAD

**Issue by vmrob** [ccbrown/needy#86]
```
$ ./b2 -j8 toolset=darwin
error: Unable to find file or target named
error:     'Waiting for other needy instances to terminate...'
error: referred to from project at
error:     '/Users/vmrob/development/myproject/needs'
```

**Comment by ccbrown** [comment]
That message should definitely not go to standard out... Will fix soon.

**Comment by ccbrown** [comment]
Resolved with 99ebfb09a5d5d0466c60cd2a6fd93bf06493a811.
